source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,669 @@
+
+.. important:: Required Field
+
+   Documents that you want to sync with an Edge Server must contain an ``_id`` 
+   field whose type varies depending on the service type: 
+   
+   - Edge Server Service: Type ``ObjectId``
+   - Device Sync Service: Type ``ObjectId``, ``String``, ``Long``, ``Int``, ``UUID``
+
+   Any documents without this field cannot sync to an Edge Server.
+
+Supported Authentication Providers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server supports a subset of authentication providers. Supported 
+authentication providers vary depending on whether you're connecting with
+Atlas Device SDK or using a MongoDB URI to connect with a MongoDB Driver 
+or tool.
+
+You must enable at least one of these providers to enable clients to connect
+to an Edge Server instance.
+
+Supported Atlas Device SDK Authentication Providers
+```````````````````````````````````````````````````
+
+- :ref:`anonymous-authentication`
+- :ref:`email-password-authentication`
+- :ref:`custom-jwt-authentication`
+- :ref:`api-key-authentication`
+
+Supported MongoDB Driver Authentication Providers
+`````````````````````````````````````````````````
+
+- :ref:`anonymous-authentication`
+- :ref:`email-password-authentication`
+
+.. _create-edge-server-instance:
+
+Create an Edge Server Instance
+------------------------------
+
+After creating an Edge Server-compatible service, you can add Edge Servers 
+to it. The service can handle multiple Edge Servers, each of which has its own 
+registration key.
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Add a server
+
+            Click :guilabel:`Add a server`. You're prompted to name your 
+            Edge Server.
+
+            The UI provides an installation script, which you can copy and 
+            paste into the terminal on the device where you want to install 
+            the Edge Server code. You also get a token, which you will use 
+            when you configure the Edge Server instance.
+
+         .. step:: Install and Configure the Edge Server code
+
+            Copy and paste the installation script into the terminal on the
+            device where you want to install the Edge Server code. For more 
+            details, refer to :ref:`edge-server-install-and-configure`.
+
+            After you configure and start the Edge Server instance, its 
+            :guilabel:`Health` status in the Edge Server dashboard changes
+            from :guilabel:`Pending` to :guilabel:`Active`.
+
+   .. tab::
+      :tabid: api
+
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/api-authenticate-instructions.rst
+
+         .. step:: Call the Create Edge Server Instance Endpoint
+
+            Call the 
+            :admin-api-endpoint:`Admin API Create an Edge Server instance <operation/adminCreateEdgeServerInstance>` 
```

Comment:
Note for reviewer: these :admin-api-endpoint: directives point to the live version of the Admin API spec: https://github.com/mongodb/snooty-parser/blob/03154c50f954010a279499a2bbf5c8bfb7b5bb58/snooty/rstspec.toml#L1535

Since the feature branch contains the API spec updates for Edge Server, all of these links will fail until we publish the feature branch. We can ignore broken link errors related to these directives as this is an expected failure.